### ISSUE TITLE
Wrong enum declaration for CGEventType

### ISSUE THREAD

**Issue by pythoneer** [servo/core-graphics-rs#70]
**Problem**

`CGEvent::new_mouse_event` is not working

**Reason**

The values for `CGEventType` are denoted as

``` 
// Mouse events.
    LeftMouseDown = 1 << 1,
    LeftMouseUp = 1 << 2,
    RightMouseDown = 1 << 3,
    RightMouseUp = 1 << 4,
    MouseMoved = 1 << 5,
    LeftMouseDragged = 1 << 6,
    RightMouseDragged = 1 << 7,
```

etc.

which seems to be wrong. I think the declaration for e.g. `NX_MOUSEMOVED` was confused with `NX_MOUSEMOVEDMASK`

If i look at like `CGEventCreateMouseEvent` from `CGEvent.h` it takes a `CGEventType` and `CGEventType` in `CGEventTypes.h`is declared like this

``` 
typedef CF_ENUM(uint32_t, CGEventType) {
  ...
  /* Mouse events. */
  ...
  kCGEventMouseMoved = NX_MOUSEMOVED,
  ...
}
```

not `NX_MOUSEMOVEDMASK`

a minimal program shows the effect. If i use `5` for mousemoved – the value of `NX_MOUSEMOVED` –  instead of the values that is declared in `CGEventType` `1<<5` it works

``` 
extern crate core_graphics;
extern crate libc;

use self::core_graphics::display::*;
use self::core_graphics::event::*;
use self::core_graphics::geometry::*;
use self::core_graphics::event_source::*;

use std::ptr;
use libc::*;


#[link(name = "ApplicationServices", kind = "framework")]
extern {
    fn CGEventCreateMouseEvent( source: CGEventSourceRef, 
                                mouseType: uint32_t,//mouseType: CGEventType,
                                mouseCursorPosition: CGPoint, 
                                mouseButton: CGMouseButton) -> CGEventRef;

    fn CGEventPost(tapLocation: CGEventTapLocation, event: CGEventRef);
}

fn main() {

    let x = 100.0;
    let y = 100.0;

    let MouseMoved = 5;

    unsafe {
        let mouse_ev = CGEventCreateMouseEvent(
            ptr::null(),
            MouseMoved, //CGEventType::MouseMoved,
            CGPoint::new(x, y),
            CGMouseButton::Left
        );

        if mouse_ev.is_null() {
            println!("ref null");
        }

        CGEventPost(CGEventTapLocation::HID, mouse_ev);     
    }
}

```

**Comment by jdm** [comment]
I agree. Want to make a PR to fix that enum?

**Comment by pythoneer** [comment]
Sure!

**Comment by pythoneer** [comment]
As the PR regarding this issue is merged into master i guess we should close it? Just as a side question, do you consider a release to crates.io anytime soon? I am interested in the functionality added by https://github.com/servo/core-graphics-rs/commit/52952115c1ddb103ebe5986de7ba5f39709fbfe0 ... you guys have quite some gap since the last one. 

best regards.

**Comment by jdm** [comment]
Sorry! It's really easy to forget about cargo releases. We often release whenever someone asks us for a new one.

**Comment by pythoneer** [comment]
what is the right way to ask for .. just create a new issue or just on #servo @irc.mozilla.org
? Don't want to pollute the issues here.  ... and i hereby ask for a new one ;)

**Comment by jdm** [comment]
Filing an issue is fine. In this case I've already opened #84 and I'll publish once it's merged.

**Comment by mbrubeck** [comment]
(for the record, this was fixed by #74.)

**Comment by jdm** [comment]
Published.
